packages/calcite-components/src/components/block/block.tsx

```diff
@@ -294,7 +325,7 @@ export class Block
     return [loading ? <calcite-scrim loading={loading} /> : null, defaultSlot];
   }
 
-  renderIcon(): VNode[] {
+  renderLoaderStatusIcon(): VNode[] {
```

Comment:
Can you make this private? For context, we'll want to explicitly use the private modifiers wherever applicable. Otherwise, Stencil will include them in types.